src/sentry/incidents/action_handlers.py

```diff
@@ -241,7 +234,7 @@ def generate_incident_trigger_email_context(project, incident, alert_rule_trigge
         # if resolve threshold and threshold type is *BELOW* then show '>'
         "threshold_direction_string": ">" if show_greater_than_string else "<",
         "status": INCIDENT_STATUS[IncidentStatus(incident.status)],
-        "status_key": INCIDENT_STATUS_KEY[IncidentStatus(incident.status)],
+        "status_key": INCIDENT_STATUS[IncidentStatus(incident.status)].lower(),
```

Comment:
does this work for `CLOSED`? looks like that becomes `resolved`

Reply:
Just double-checked `/src/sentry/incidents/models.py`: 
```
INCIDENT_STATUS = {
    IncidentStatus.OPEN: "Open",
    IncidentStatus.CLOSED: "Resolved",
    IncidentStatus.CRITICAL: "Critical",
    IncidentStatus.WARNING: "Warning",
}
```
It'll do the right thing.